From 4913b3571ab259984a8b0ff60043fda2c6de6558 Mon Sep 17 00:00:00 2001
From: Jason Liu <jason.hui@linaro.org>
Date: Wed, 14 Dec 2011 19:36:46 +0800
Subject: [PATCH 0716/2463] Revert "ENGR00155612-3 [mx6q]add delay after cmd6
 for eMMC compatibility"

This reverts commit dee9bc3d9c98bc45ad42960a6650dffc66140d19.
---
 drivers/mmc/core/mmc_ops.c |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/drivers/mmc/core/mmc_ops.c b/drivers/mmc/core/mmc_ops.c
index b11876c..845ce7c 100644
--- a/drivers/mmc/core/mmc_ops.c
+++ b/drivers/mmc/core/mmc_ops.c
@@ -397,7 +397,6 @@ int mmc_switch(struct mmc_card *card, u8 set, u8 index, u8 value,
 	err = mmc_wait_for_cmd(card->host, &cmd, MMC_CMD_RETRIES);
 	if (err)
 		return err;
-	mdelay(1);
 
 	/* Must check status to be sure of no errors */
 	do {
-- 
1.7.7.4

